### PR TITLE
Updates README 

### PR DESCRIPTION
**Pull request by kierangillen**
- Adds README clarification for emission deployment

#trivial

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,13 +94,13 @@ We vendor some data from other repositories that you will sometimes need to upda
 
 ### Deploying Emission
 
-**Note:** Deploys are mostly automated using `yarn release patch|minor|major`; the following instructions should be valid but take a look at [this issue](https://github.com/artsy/emission/issues/1077#issuecomment-401128949) if things break.
+**Note:** Deploys are mostly automated using `yarn release patch|minor|major`; the following instructions should be valid but take a look at [this issue](https://github.com/artsy/emission/issues/1077#issuecomment-401128949) if things break. To release an update to bump Eigen's Emission version, you must use either a major or minor release.
```
Comment (alloy) [inline README.md]: I’m not sure I understand what this is about, it’s fine to do a patch release for inclusion in Eigen. What gives?

File: README.md [hunk 2]
```diff
@@ -94,13 +94,13 @@ We vendor some data from other repositories that you will sometimes need to upda
...
 
 **Note 2:** If you're updating React Native, you're gonna have to do a lot of this stuff manually. See the makefile/alloy/orta for more info.
 
 1. Don't change the `package.json` version.
 1. For non-beta releases, use either `yarn release patch`, `yarn release minor`, or `yarn release major`. For
-   beta releases use `yarn release prerelease`.
+   beta releases, use `yarn release prerelease`. Beta releases are usually reserved for testing major releases before deployment.
```
Comment (alloy) [inline README.md]: Perhaps we should remove the word “beta” and keep it to “pre-releases”, as the command does. Then we can say that pre-releases are meant for testing finalized versions that will still receive QA.

File: README.md [hunk 3]
```diff
@@ -96,12 +96,13 @@ We vendor some data from other repositories that you will sometimes need to upda
 
 **Note:** Deploys are mostly automated using `yarn release patch|minor|major`; the following instructions should be valid but take a look at [this issue](https://github.com/artsy/emission/issues/1077#issuecomment-401128949) if things break.
 
-**Note 2:** If you're updating React Native, you're gonna have to do a lot of this stuff manually. See the makefile/alloy/orta for more info.
+**Note 2:** If you're updating React Native, you're going to have to do a lot of this stuff manually. See the makefile or alloy for more info.
```
Comment (alloy) [inline README.md]:
```suggestion
**Note 2:** If you're updating React Native, you're going to have to do a lot of this stuff manually. See the makefile, ashfurrow, or alloy for more info.
```

File: README.md [hunk 4]
```diff
@@ -96,12 +96,13 @@ We vendor some data from other repositories that you will sometimes need to upda
...
 
 1. Don't change the `package.json` version.
-1. For non-beta releases, use either `yarn release patch`, `yarn release minor`, or `yarn release major`. For
-   beta releases use `yarn release prerelease`.
-1. If integrating in Eigen consult the docs [on updating Emission](https://github.com/artsy/eigen/blob/master/docs/updating_emission.md)
+2. For non-pre-releases, use either `yarn release patch`, `yarn release minor`, or `yarn release major`. For
+   pre-releases, use `yarn release prerelease`. Pre-releases are usually reserved for testing major releases before deployment.
+3. After the release has finished compiling, a PR will be automatically created in Metaphysics, which can be self assigned and merged.
+4. If integrating in Eigen consult the docs [on updating Emission](https://github.com/artsy/eigen/blob/master/docs/updating_emission.md)
```
Comment (alloy) [inline README.md]: FYI If you only use `1.` then when markdown gets rendered it automatically increases the number, which means you can move items around in the list without having to change numbers the whole time.

File: README.md [hunk 5]
```diff
@@ -96,12 +96,13 @@ We vendor some data from other repositories that you will sometimes need to upda
...
-**Note 2:** If you're updating React Native, you're gonna have to do a lot of this stuff manually. See the makefile/alloy/orta for more info.
+**Note 2:** If you're updating React Native, you're going to have to do a lot of this stuff manually. See the makefile or alloy for more info.
 
 1. Don't change the `package.json` version.
-1. For non-beta releases, use either `yarn release patch`, `yarn release minor`, or `yarn release major`. For
-   beta releases use `yarn release prerelease`.
-1. If integrating in Eigen consult the docs [on updating Emission](https://github.com/artsy/eigen/blob/master/docs/updating_emission.md)
+2. For non-pre-releases, use either `yarn release patch`, `yarn release minor`, or `yarn release major`. For
+   pre-releases, use `yarn release prerelease`. Pre-releases are usually reserved for testing major releases before deployment.
+3. After the release has finished compiling, a PR will be automatically created in Metaphysics, which can be self assigned and merged.
```
Comment (alloy) [inline README.md]: Maybe add some detail about what the PR contains.

File: README.md (hunk 4 above)
Comment (kierangillen) [reply README.md]: Oh wow, didn't know, awesome.

File: README.md (hunk 5 above)
Comment (kierangillen) [reply README.md]: 👍

File: README.md (hunk 3 above)
Comment (kierangillen) [reply README.md]: Ash aint getting off that easy.

Comment (alloy) [reply README.md]: lol

File: README.md [hunk 6]
```diff
@@ -96,11 +96,12 @@ We vendor some data from other repositories that you will sometimes need to upda
 
 **Note:** Deploys are mostly automated using `yarn release patch|minor|major`; the following instructions should be valid but take a look at [this issue](https://github.com/artsy/emission/issues/1077#issuecomment-401128949) if things break.
 
-**Note 2:** If you're updating React Native, you're gonna have to do a lot of this stuff manually. See the makefile/alloy/orta for more info.
+**Note 2:** If you're updating React Native, you're going to have to do a lot of this stuff manually. See the makefile, ashfurrow or alloy for more info.
```
Comment (alloy) [inline README.md]: No Oxford comma??? 😱

**Comment by alloy** [comment]
Great, thanks! 🚀 

#MergeOnGreen

File: README.md (hunk 6 above)
Comment (kierangillen) [reply README.md]: I'm a minimalist at heart

**Pull request merged**